cmake/SetupCompiler.cmake

```diff
@@ -110,6 +110,12 @@ if(APPLE)
     add_definitions(-DHAS_CORE_ML)
     set(HAS_CORE_ML TRUE)
   endif()
+  
+  # MLCompute is only present on macOS 11.0 or higher.
+  if(NOT TC_BASE_SDK_VERSION VERSION_LESS 10.16)
```

Comment:
Should this be `11.0` rather than `10.16`? Or does the comment on the line above it need to be updated?

Reply:
Changing to 10.16 for now.

Reply:
i think comparing for `>= 10.16` makes sense (as opposed to `11.0`). is there any reason not to write this more clearly as:

```
if (TC_BASE_SDK_VERSION GREATER_EQUAL 10.16)
```

Reply:
sorry, this should be `VERSION_GREATER_EQUAL` not `GREATER_EQUAL`